src/schema/rules/files/raw/meg.yaml

```diff
@@ -3,19 +3,34 @@ meg:
   suffixes:
     - meg
   extensions:
-    - / # corresponds to BTi/4D data
-    - .ds/
+    # / corresponds to BTi/4D data
     - .json
-    - .fif
-    - .sqd
-    - .con
+    - [/, .ds/, .fif]
+    - [.sqd, .con]
+    - .ave # mentioned once in the appendix
+    # .raw and .raw.mhd work as a pair
     - .raw
-    - .ave
-    - .mrk
+    - .raw.mhd
+    # .kdf with possible additional files .chn, .trg and _digitizer.txt (see other suffix group below)
     - .kdf
-    - .mhd
```

Comment:
according to the appendix example it is supposed to be `.raw.mhd` and not `.raw` and but the text above the example that we should have `.raw` and `.mhd`

Reply:
@guiomar Can you confirm or delegate someone to confirm the correct file extensions for MEG?

Reply:
perhaps @robertoostenveld can be of help here with his MEG knowledge.

Reply:
Here's the description I have in #919 for the `.raw` extension:

> When produced by a KIT / Yokogawa / Ricoh MEG system, this file contains trial-based evoked fields.
>
> When produced by an ITAB-ARGOS153 system, this file contains raw data and is generated along with an associated binary header file  with the `.mhd` extension.

So  I think`.raw` can either be a one-off extension or a paired extension with `.mhd`, and `.mhd` only appears as a pair with a `.raw`.

Reply:
FWIW the validator [uses `.raw.mhd`](https://github.com/bids-standard/bids-validator/blob/d76a498e04cf3f7f5b36bdd92f230bb328ea093a/bids-validator/bids_validator/rules/file_level_rules.json#L476): `(.(?!\\.(sqd|con|fif|raw|raw\\.mhd|trg|kdf|chn)$))`.

Searching through OpenNeuro, I cannot find any datasets with either `.mhd` or `.raw.mhd`.

Reply:
Here's where I got my description from: https://bids-specification.readthedocs.io/en/v1.5.0/99-appendices/06-meg-file-formats.html#itab

The appendix definitely says they're separate files.

Reply:
Oh sorry, maybe I misunderstood the text.